src/fetch/createFetchSharedClient.ts

```diff
@@ -43,7 +43,10 @@ const getTimeoutUtils = (timeoutDurationMs: number | undefined) => {
 
   const controller = new AbortController();
   const { signal } = controller;
-  const timeoutId = setTimeout(() => controller.abort(), timeoutDurationMs);
+  const timeoutId = setTimeout(
+    () => controller.abort(new Error(`timeout of ${timeoutDurationMs}ms exceeded`)),
```

Comment:
j'en ai profité pour changer le message par défault qui était : `This operation was aborted`, c'est quand même plus clair comme ça. (j'ai aligné la syntaxe sur celle d'axios)